source/users.puml

```diff
@@ -0,0 +1,19 @@
+@startuml
+
+package "Level 0 data" as L0 {
+  (Consume data) as (CD0)
+}
+
+package "Level 1 data" as L1 {
+  (Produce data) as (PD1)
+  (Consume data) as (CD1)
+}
+
+actor "L0 consumer" as L0C
```

Comment:
Should this also be labelled as a L1 producer?

Reply:
unless it will be too much text? What about the following:
```suggestion
actor "L0 consumer / L1 producer" as L0C
```